src/main/java/org/skriptlang/skriptworldguard/elements/expressions/ExprRegions.java

```diff
@@ -0,0 +1,78 @@
+package org.skriptlang.skriptworldguard.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.World;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skriptworldguard.worldguard.RegionUtils;
+import org.skriptlang.skriptworldguard.worldguard.WorldGuardRegion;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Name("All Regions")
+@Description("Get all regions from all worlds or from a specific world.")
+@Examples({
+		"send all regions",
+		"set {_regions::*} to the regions in world \"world\""
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("WorldGuard 7")
+public class ExprRegions extends SimpleExpression<WorldGuardRegion> {
+
+	static {
+		Skript.registerExpression(ExprRegions.class, WorldGuardRegion.class, ExpressionType.SIMPLE,
```

Comment:
```suggestion
		Skript.registerExpression(ExprRegions.class, WorldGuardRegion.class, ExpressionType.COMBINED,
```
due to presence of expression (even if optional)